Frontend/src/test/java/com/harmonia/client/PublicMessageClientTest.java

```diff
@@ -83,8 +83,8 @@ public void editPublicMessageTest() {
         request.setPmessageId(3L);
         request.setMessageText("Test updating message!");
         when(restTemplate.exchange(anyString(), any(HttpMethod.class), any(HttpEntity.class), Mockito.<Class<PublicMessagePO>>any(), anyMap())).thenReturn(new ResponseEntity<>((request), HttpStatus.OK));
-        PublicMessagePO responsePublicMessagePO = publicMessageClient.editPublicMessage(request);
-        assertEquals(request.getMessageText(), responsePublicMessagePO.getMessageText());
+        String responsePublicMessagePO = publicMessageClient.editPublicMessage(request).getBody();
+        assertEquals("Message has been edited", responsePublicMessagePO);
```

Comment:
Make sure this is what you want. I think it should return the PublicMessagePO. It is easier to handle.

Reply:
I don't exactly like this solution either, but would have to make changes to the backend to make it return the PublicMessagePO and i don't know how to make it happen. This will work as a band-aid solution for tomorrows presentation and can be corrected afterwards.